app/models/further_information_request.rb

```diff
@@ -66,5 +66,17 @@ def after_expired(user:)
     DeclineQTS.call(application_form:, user:) unless application_form.withdrawn?
   end
 
+  def after_reviewed(user:)
+    # implement logic after this requestable has been reviewed
+    items.each do |item|
+      next unless item.work_history_contact?
```

Comment:
```suggestion
    items.select(&:work_history_contact?).each do |item|
```

I think this is slightly more conventional than iterating over every item.